lib/liquid/tag.rb

```diff
@@ -17,8 +17,6 @@ def disable_tags(*tags)
         disabled_tags.push(*tags)
       end
 
-      private :new
```

Comment:
Do you know what was the intent behind making `new` private?

Reply:
I can have a look back through, but I'm assuming this is to avoid creating tags outside of Liquid.